app/assets/javascripts/application.js

```diff
@@ -279,6 +280,10 @@ function autoSync() {
   hasMarkedRows() || sync()
 }
 
+function clearFilters(){
+  Turbolinks.visit("/");
+}
+
```

Comment:
Just a nit we need a space before the '{'  :] does the 'visit' remove the urls query params too?

Reply:
Yes it does.

Reply:
And space added 👍